dockstore-integration-testing/src/test/java/io/dockstore/client/cli/BasicIT.java

```diff
@@ -609,6 +613,19 @@ public void testPublishAndUnpublishTool() {
         publishAndUnpublishToolHelper("quay.io/dockstoretestuser/quayandgitlab");
     }
 
+    @Test
+    public void testPublishToolEvents() {
+        Assert.assertEquals("There should be no publish events", 0, testingPostgres.getPublishEventCount());
+        Assert.assertEquals("There should be no publish events", 0, testingPostgres.getUnpublishEventCount());
```

Comment:
unpublished

Reply:
Good catch, fixed in 698264e5f327e95583565c5419cd8bca13748c6c